packages/itmat-commons/src/models/log.ts

```diff
@@ -68,16 +68,13 @@ export enum LOG_ACTION {
     createFieldCurationJob = 'CREATE_FIELD_CREATION_JOB',
     createDataCurationJob = 'CREATE_DATA_CURATION_JOB',
     getDataRecords = 'GET_DATA_RECORDS',
-    getOntologyTree = 'GET_ONTOLOGY_TREE',
```

Comment:
You cannot remove these here because past records in the database might still exist and be valid ⚠️